src/features/create-quiz/create-quiz-question-list/question-list.tsx

```diff
@@ -14,9 +15,18 @@ export const QuestionList: React.FC<QuestionListProps> = observer(() => {
         {QuizStore.questions.map((question, index) => (
           <div key={index}>
             <QuestionItem
-              questionName={`Question ${question.question}`}
+              questionName={`${
+                question.question?.questionName
```

Comment:
This type of ternary check can be simplified with the syntax: `question.question?.questionName || "Untitled question"`. This syntax is often used to give default values in case the main value is falsy.